install_config/certificate_customization.adoc

```diff
@@ -108,20 +108,22 @@ servingInfo:
   bindNetwork: tcp4
   certFile: master.server.crt <1>
   clientCA: ""
-  keyFile: master.server.key <1>
+  keyFile: master.server.key <2>
   maxRequestsInFlight: 0
   requestTimeoutSeconds: 0
   namedCertificates:
-    - certFile: wildcard.example.com.crt <2>
-      keyFile: wildcard.example.com.key <2>
+    - certFile: wildcard.example.com.crt <3>
+      keyFile: wildcard.example.com.key <4>
       names:
         - "openshift.example.com"
   metricsPublicURL: "https://metrics.os.example.com/hawkular/metrics"
 
 ----
 
-<1> Path to certificate and key files for the CLI and other API calls.
-<2> Path to certificate and key files for the web console.
+<1> Path to the certificate file for the CLI and other API calls.
+<2> Path to the key file for the CLI and other API calls.
+<3> Path to the certificate file for the web console. If necessary, be sure to concatenate all of the required files that form your xref:../install_config/certificate_customization.adoc#using-certificate-chains[certificate chain] for the certificate file that is provided to the `certFile` parameter.
```

Comment:
Does it make sense to

s/be sure to//

for conciseness?

Reply:
These aren't for the console. These are for the apiserver. It looks like this entire section is a bit out of date starting at the point at the console was moved to pods.
@spadgett Can you please confirm?

Reply:
In 3.11, the master still proxies the console pods, so the public master hostname and console hostname are the same. (That's no longer true in 4.x, however, where console is behind a route.)

Reply:
Thanks @spadgett. So is this section still accurate for 3.11, or do we need to make some changes here?

Reply:
I'm not sure to be honest since this is master config and the master team owns the proxy in 3.11. @deads2k do you know?

Reply:
Ahh, so in 3.11 even though it was a pod it was proxied via api. ? If so then this seems correct but I'd make sure to say something about "Public API" in addition to console here.

Reply:
> Ahh, so in 3.11 even though it was a pod it was proxied via api. ?

That's right.